INSTALL.md

```diff
@@ -1,45 +1,42 @@
+7. Do not update _Gradle_ version used in the project, in case you are prompted to update it.
+8. Download the OBB file from the latest GitHub release and follow the instructions from there itself.
 
 #### Note:
-1. The _mainapp_, _seaworld_ and _bookviewer_ projects build correctly on Mac OS only. Other applications build on both Mac and Windows OS.
-2. If any of the applications throw APK signing error then either comment the signing configuration from the app's gradle file or create your own _keystore_ file with the help of steps mentioned [here](https://developer.android.com/studio/publish/app-signing#generate-key).
-3. _KitkitSchoolLogger_ module is among other projects.
-4. Set the build variant to __debug__ for all project apps except for the _mainapp_ which is described in point 4 of FAQs below.
-5. There is no need to install _cocos_ application separately for these projects.
+1. If any project throws APK signing error during build process then either comment the signing configuration from the app's gradle file or create your own _keystore_ file with the help of steps mentioned [here](https://developer.android.com/studio/publish/app-signing#generate-key).
+2. _kitkitschoollogger_ module is shared between _pehlaschool_ and _pehlaschoollibrary_.
+4. There is no need to install _cocos_ application separately for these projects.
 
 ## FAQs
 1. There is a cross mark on the app cofiguration button. How do I fix that?
 
     >The default/main activity of the application is not set for the project. Either go to _app/build/outputs/apk_ folder and install the APK which is generated right after the build or, set the start activity in the app configuration settings of the project.
 
 2. How to enable the _Build variant_ option in the _Build_ menu?
-	>The _Build variant_ option is enabled only when the module level gradle file is opened.
+	>The _Build variant_ option is enabled only when the module's gradle file is opened.
 
 3. The application throws _keystore_ file not found error. How do I fix this?
 
 	>_keystore_ file is used to digitally sign an Android application and hence, it is not provided with the project. One must create their own _keystore_ file using the [Android keytool](https://developer.android.com/studio/publish/app-signing) which comes with the Android Studio itself in order to sign the APK.
 
-4. How to fix the the mainapp application which throws a run time error related to the package name _com.enuma.xprize_?
+4. How to fix the the pehlaschool application which throws a run time error related to the package name _com.maq.pehlaschool_?
 	> This error is caused by incorrect build variant setting for the application. Kindly, refer to the below table which has the build variant settings.
 	> 
 	> |App name| Build variant|
 	> |--------|--------------|
-	> |KitkitSchool|generalDebug-armeabi|
-	> |kitkitlogger|release|
+	> |PehlaSchool|hindiDebug/hindiRelease|
```

Comment:
Add other languages build variant

Reply:
This is to avoid any issue with the OBB file. However, your suggestion is covered in the last step of installation steps.